pages/_app.js

```diff
@@ -1,16 +1,56 @@
-import '../styles/globals.scss'
-import Head from 'next/head'
+import "../styles/globals.scss";
+import Head from "next/head";
+import { appWithI18Next, useSyncLanguage } from "ni18n";
+import { ni18nConfig } from "../ni18n.config";
+import { useTranslation } from "react-i18next";
+import { useEffect } from "react";
+import Cookies from 'js-cookie';
 
 function MyApp({ Component, pageProps }) {
-  return <>
-    <Head>
-      <title>Meu Arteliê</title>
-      <meta name="viewport"  key="viewport" content="initial-scale=1.0, width=device-width" />
-      <meta name="description" key="description" content="Descubra as mandalas e seus significados. Aprenda a desegnar, colorir e manifestar." />
-    </Head>
-       
-    <Component {...pageProps} />
-  </>
+  const { t, i18n } = useTranslation();
+
+  // useSyncLanguage(locale);
+
+  const changeLanguage = () => {
+    const locale = Cookies.get('MY_LANGUAGE');
+    const urlParams = new URLSearchParams(window.location.search);
+    const language = urlParams.get("lang");
+    const browserLang = navigator.language || navigator.userLanguage; 
+
+    if (language) {
```

Comment:
this was supposed to use [detectors](https://github.com/i18next/i18next-browser-languageDetector)